spec/policies/person_policy_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe PersonPolicy do
+  describe '#admin?' do
+    context 'the person is not signed in' do
+      subject { described_class.new(nil_person, 'any object') }
+      let(:nil_person) { nil }
+      it 'returns false' do
+        expect(subject.admin?).to eq(false)
+      end
+    end
+
+    context 'the person is a non admin' do
+      subject { described_class.new(non_admin, 'any_object') }
```

Comment:
When we're describing a method it's usually nicer to set the subject once and then chance something particular about it in each context.

```ruby
describe "#admin?" do
  subject { described_class.new(person, "object").admin? }
  
  context "when the person is nil" do
    let(:person) { nil }
    it { is_expected.to eq false }
  end

  context "when the person is signed" do
    context "and is not an admin" do
      let(:person) {}
      it { is_expected.to eq false }
    end

    context "and is an admin" do
      let(:person) {}
      it { is_expected.to eq true }
    end
  end
end
```

Additionally, you'll note that my subject in the example above is the method call itself. That way you don't have to do `subject.admin?` in every assertion.